github/reactions.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import "fmt"
+
+// ReactionsService provides access to the reactions-related functions in the
+// GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/reactions/
+type ReactionsService struct {
+	client *Client
+}
+
+// Reaction represents a GitHub reaction.
+type Reaction struct {
+	// ID is the Reaction ID.
+	ID     *int `json:"id,omitempty"`
+	UserID *int `json:"user_id,omitempty"`
+	// Content is the type of reaction.
+	// Possible values are:
+	//     "+1", "-1", "laugh", "confused", "heart", "hooray".
+	Content *string `json:"content,omitempty"`
+}
+
+// ReactionsSummary represents a summary of GitHub reactions.
+type ReactionsSummary struct {
+	TotalCount *int    `json:"total_count,omitempty"`
+	Plus1      *int    `json:"+1,omitempty"`
+	Minus1     *int    `json:"-1,omitempty"`
```

Comment:
I like both of those options too. It doesn't seem that I have a preference for either one, because I can't choose what to say. :)

Maybe I have an ever so slight preference towards `PlusOne`/`MinusOne` over `ThumbsUp`/`ThumbsDown`, because I tend to think of those 2 emoji in my mind as "plus one/minus one", I type them as `:+1:`, and it's slightly shorter. Plus more different/easier to tell apart.